src/Resources/skeleton/web/app.php

```diff
@@ -21,6 +22,19 @@
 /** @var Composer\Autoload\ClassLoader */
 $loader = require __DIR__.'/../vendor/autoload.php';
 
+if (file_exists(__DIR__.'/../.env')) {
+    (new Dotenv())->load(__DIR__.'/../.env');
+}
+
+// see https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/public/index.php#L27
+if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? false) {
```

Comment:
This is a rather unconventional piece of code. 😄 I guess we should at least use `?? null` instead of `?? false`, shouldn't we?

Reply:
I see you have copied it from Symfony …